ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -50,25 +51,20 @@ public static class ChangeFeedProcessor
             string eventHubNamespaceConnection = ConfigurationSettings.AppSettings["EventHubNamespaceConnection"];

```

Comment:
nit:  The product name is `Event Hubs` of which there is one ore more `Event Hub` instances.   We should be using this terminology and respecting capitalization.